### DISCUSSION TITLE
画布放大后线条失真问题

### DISCUSSION THREAD

**Discussion by iamjichao** [hizzgdev/jsmind#598] [General]
画布放大后，线条失真了：
<img width="174" alt="放大失真" src="https://github.com/hizzgdev/jsmind/assets/24667185/df293db7-f271-4a53-a316-4e2f56e6fd93">

**Comment by hizzgdev** [comment]
是的，把 view.engine 换成 svg 就可以了。你试试？
https://hizzgdev.github.io/jsmind/docs/zh/2.options.html

**Reply by iamjichao** [reply]
好的，已解决。感谢！
